backend/s3/s3.go

```diff
@@ -569,6 +583,7 @@ type Options struct {
 	LocationConstraint   string        `config:"location_constraint"`
 	ACL                  string        `config:"acl"`
 	ServerSideEncryption string        `config:"server_side_encryption"`
+	SSEKMSKeyId          string        `config:"sse_kms_key_id"`
```

Comment:
If you change this one to `SSEKMSKeyID` then the linter should be satisfied (cross fingers!).